content/en/docs/concepts/workloads/controllers/statefulset.md

```diff
@@ -298,8 +298,7 @@ described [above](#deployment-and-scaling-guarantees).
 `Parallel` pod management tells the StatefulSet controller to launch or
 terminate all Pods in parallel, and to not wait for Pods to become Running
 and Ready or completely terminated prior to launching or terminating another
-Pod. This option only affects the behavior for scaling operations. Updates are not
-affected.
+Pod. This option only affects the behavior for scaling operations.
```

Comment:
I don't think this change actually affects the meaning of the paragraph (it's removing a clarification). If we intended to change the meaning, I think we should do a different rewrite.

Reply:
This is indeed expected change as described in https://github.com/kubernetes/kubernetes/issues/117071 and fixed in https://github.com/kubernetes/kubernetes/pull/117865. With that fix we should be explicit that this operation applies to both scaling and pod creation.

Reply:
There will be followup changes after we push forward the mechanism described in https://github.com/kubernetes/enhancements/issues/961 which currently blocked on monitoring requirements to be able to progress with that effort.

Reply:
This is needful as it is described in https://github.com/kubernetes/kubernetes/issues/117071 and was fixed in https://github.com/kubernetes/kubernetes/pull/117865 Thi is expected changes

Reply:
What's the difference in meaning between the two options (before and after) here?

```diff
-This option only affects the behavior for scaling operations. Updates are not affected.
+This option only affects the behavior for scaling operations.
```

To me they have the same meaning, although the original includes a clarifying statement.

Reply:
@sftim  got your point. Yes we should not ignore the last sentence as well like Updates are not affected.

The sentence is mandatory.

Reply:
Suggestions for improving this sentence 
```suggestion
This option only affects the behavior of scaling operations and does not impact updates.
```

Reply:
I this this  sentence will look more better .

Reply:
From what I have tested, this option affects updates, too. See https://github.com/kubernetes/website/issues/47085. Maybe I am not able to correctly understand what we mean by updates here.